core/modules/system/config/system.core.json

```diff
@@ -46,6 +46,7 @@
     "maintenance_mode_message": "[site:name] is currently under maintenance. We should be back shortly. Thank you for your patience.",
     "maintenance_page_maximum_age": 10,
     "maintenance_theme": null,
+    "menu_breakpoint": "default",
```

Comment:
A part of me is a little uneasy adding a setting that is in an optional module to `system.core.json`. Although I don't like having a settings file with only one setting in it, but I think moving this to a new `menu.settings.json` file would be appropriate, and might give us a place to put menu settings in the future, especially as we're already making a dedicated settings form in menu module with this PR.